src/Entity/BaseBlock.php

```diff
@@ -23,11 +23,18 @@ abstract class BaseBlock extends Block
     public function prePersist(): void
     {
         $this->createdAt = new \DateTime();
-        $this->updatedAt = new \DateTime();
+
+        $this->preUpdate();
```

Comment:
I dunno if it's not better to duplicate the logic.

If someone override the preUpdate method
```
public function preUpdate(): void
{
    parent::preUpdate();

    // do things
}
```
he may not want to execute them in the prePersist too.

Reply:
Changed.